drivers/digit7seg/doc.txt

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (C) 2024 Orange
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+
+@defgroup    drivers_digit7seg Generic 7 segments driver
+@ingroup     drivers_display
+@brief       Device driver for less than 5 digits of 7 segments without IC
+
+## About
+
+This driver was developed to works with [5461AS](http://www.topliteusa.com/uploadfile/2014/0825/A-5461AS.pdf)
+in a way that all 4 digits (and less) with 7 segments without integrated controller
+can be controlled into [RIOT-OS](https://github.com/RIOT-OS/RIOT).
```

Comment:
```suggestion
can be controlled from RIOT-OS.
```

this will become part of doc.riot-os.org, so no need to link to the RIOT github page I'd say.